graphql/queryResolvers/addStar.ts

```diff
@@ -0,0 +1,31 @@
+import db from '../../helpers/dbload'
+import { LoggedRequest } from '../../@types/helpers'
+import { Star as StarType } from '../../@types/lesson'
+
+const { Star } = db
+
+export const addStar = async (
+  _parent: void,
+  arg: StarType,
+  ctx: { req: LoggedRequest }
+) => {
+  const { req } = ctx
+  try {
+    const { studentId, mentorId, lessonId } = arg
+    if (!studentId) {
+      throw new Error('Missing or invalid studentId')
+    }
+    if (!mentorId) {
+      throw new Error('Missing or invalid mentorId')
+    }
+    if (!lessonId) {
+      throw new Error('Missing or invalid lessonId')
+    }
+
+    await Star.create(arg)
```

Comment:
1. get existing star for the student / lessonId.
2. if exist, replace
2. if not exist, create new.

every student can only give 1 star per lesson.